File: web/angular/src/app/classes/orderTicket.ts
import { Time } from "@angular/common";

export class orderTicket{
    constructor(public id?:number, public userId?:number, public fullName?:string, public destination?:string
        ,public dateOfTrip?:Date, public orderTime?:Time , public hours?:number , public tripId?:number, public tickets?:number) {       
    }
    }

File: C#/Dal/Dto/Classes/OrderTicket.cs
using System;
using System.Collections.Generic;
namespace Dto.Classes;

public partial class OrderTicket
{
    public int Id { get; set; }

    public int? UserId { get; set; }

    public string? fullName { get; set; }

    public string? destination {  get; set; }
   /*תאריך הזמנת הטיול*/
    public DateTime? Date { get; set; }
    /*תאריך יציאה לטיול*/
    public DateTime? DateOfTrip { get; set; }
    /*שעת ההזמנה */
    public TimeSpan? OrderTime { get; set; }

    public int? Hours { get; set; }

    public int? TripId { get; set; }

    public int? Tickets { get; set; }

}
